gapic-generator-java/DEVELOPMENT.md

```diff
@@ -155,6 +155,39 @@ To generate a production GAPIC API:
+      ```
+   4. Run the protoc command.
+      
+      It shows nothing because it's waiting for a debugger to attach.
+
+2. In the generator, add [Remote JVM debug configuration](https://www.jetbrains.com/help/idea/tutorial-remote-debug.html) for the generator in Intellij.
```

Comment:
```suggestion
2. Add [Remote JVM debug configuration](https://www.jetbrains.com/help/idea/tutorial-remote-debug.html) for the generator in Intellij.
```

I didn't get the meaning of "In the generator".

Reply:
Just to guide the reader to the generator repo. Remove it also fine.

Reply:
Then "In the IntelliJ that has gapic-generator-java"

Reply:
done

Reply:
done